django/db/backends/oracle/introspection.py

```diff
@@ -7,6 +8,8 @@
 )
 from django.utils.deprecation import RemovedInDjango21Warning
 
+FieldInfo = namedtuple('FieldInfo', FieldInfo._fields + ('is_autofield',))
```

Comment:
I think I mentioned this before -- I find this name rebinding a bit odd. I'd like it better if the `FieldInfo` from `base.introspection` was imported as `BaseFieldInfo`.